oci/oci_linux.go

```diff
@@ -0,0 +1,57 @@
+// +build linux
+
+package oci
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/containerd/cgroups"
+	"github.com/kubernetes-incubator/cri-o/utils"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sys/unix"
+)
+
+func (r *Runtime) createContainerPlatform(c *Container, cgroupParent string, pid int) error {
+	// Move conmon to specified cgroup
+	if r.cgroupManager == SystemdCgroupsManager {
+		logrus.Infof("Running conmon under slice %s and unitName %s", cgroupParent, createUnitName("crio-conmon", c.id))
+		if err := utils.RunUnderSystemdScope(pid, cgroupParent, createUnitName("crio-conmon", c.id)); err != nil {
+			logrus.Warnf("Failed to add conmon to systemd sandbox cgroup: %v", err)
+		}
+	} else {
```

Comment:
Now that this is in a function, I'd consider ending the previous block with `return nil` so we can drop this `else` and flatten the next block.  That would be:

```go
if r.cgroupManager == SystemdCgroupsManager {
  …
  return nil
}
control, err := cgroups.New(cgroups.V1, cgroups.StaticPath(filepath.Join(cgroupParent, "/crio-conmon-"+c.id)),
```

Reply:
fair